### PR TITLE
[EDU-4489] feat: add cli purge docs

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: https://aziontech.atlassian.net/browse/EDU-4489

### CODE REVIEW

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Azion CLI purge
+description: Azion CLI Purge allows you to instantly delete the cache of your content at Azion..
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx:3]:
```suggestion
description: Azion CLI purge allows you to instantly delete the cache of your content at Azion..
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/purge/purge.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Azion CLI purge
+description: Azion CLI Purge permite que você realize o purge do cache do seu conteúdo na Azion.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/purge/purge.mdx:3]:
```suggestion
description: Azion CLI purge permite que você realize o purge do cache do seu conteúdo na Azion.
```

**Review by lfsigreja — APPROVED** [review]
lgtm

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Azion CLI purge
+description: Azion CLI purge allows you to instantly delete the cache of your content at Azion..
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx:3]: description: Azion CLI purge allows you to instantly delete the cache of your content at Azion.

**Review by gabriel-alves-azion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx (hunk 3 above)
Comment (gabriel-azion) [reply src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx:3]:
```suggestion
description: Azion CLI purge allows you to instantly delete the cache of your content at Azion.
```

**Pull request merged**